### PR TITLE
Feat/cancellation form

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         |no
| New feature?     | yes
| Breaking change? |no
| Tickets          | MANAGER-6023
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/delete.routing.js [hunk 1]
```diff
@@ -1,37 +1,78 @@
...
+        service,
+      ) => () => {
+        atInternet.trackClick({
+          name: `autorenew::${kebabCase(service.serviceType)}::delete::confirm`,
+          type: 'action',
+          chapter1: 'dedicated',
+          chapter2: 'account',
+          chapter3: 'billing',
+        });
+        service.setForResiliation();
```
Comment (Jisay) [inline packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/delete.routing.js:57]: `setFormResiliation` ?

**Review by Jisay — APPROVED** [review]

**Review by cbourgois — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/delete.routing.js (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/delete.routing.js:57]: No `setForResiliation` is a method on the `BillingService` class to set all `renew` fields correctly for resilitation

**Review by antleblanc — DISMISSED** [review]

**Comment by marie-j** [comment]
Rebase to fix conflicts and minor improvements

**Comment by antleblanc** [comment]
> Rebase to fix conflicts and minor improvements

There are new conflicts :see_no_evil:! Could you please rebase?

Thank you!

**Comment by marie-j** [comment]
@antleblanc Solved 🙂

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**